client/src/hooks/useFilters.ts

```diff
@@ -0,0 +1,33 @@
+import { useState, useCallback } from 'react';
+import { SORT_DIRECTION } from '../components/Filters/variables';
+
+export const useFilters = () => {
+  const [filter, setFilter] = useState({
+    page: 1,
+    sortBy: 'popularity',
+    sortDirection: SORT_DIRECTION.DESC,
+    includeAdult: true,
+  });
+
+  const setPage = useCallback((page: number) => {
+    setFilter({
+      ...filter,
+      page,
+    });
+  }, [filter]);
+
+  const setFiltering = useCallback((filterFields: any) => {
```

Comment:
create type for this
```suggestion
  const setFiltering = useCallback((filterFields: ...) => {
```